locales/es/about.json

```diff
@@ -25,7 +25,7 @@
       "Integración de QuranReflect: interactúe con una comunidad global a través de reflexiones y conocimientos compartidos por académicos e individuos.",
       "Seguimiento del progreso de lectura y objetivos: realice un seguimiento de sus objetivos diarios y su historial de lectura.",
       "API para desarrolladores: acceso gratuito a contenido y funciones para impulsar aplicaciones islámicas e I+D.",
-      "Y mucho más, Alhamdullilah."
+      "Y mucho más, Alhamdulillah."
```

Comment:
@zonetecde we only need to change English language when it comes to translations. We use Lokalise to translate the rest. Flow is PR adds/updates/delete English language -> during deployment we push those changes to Lokalise -> QF's translation team add/update/delete translations of the rest of the language -> Person responsbile for deployment pulls again from Lokalise to the code base.